app/views/shared/_footer.html+new_navigation.erb

```diff
@@ -0,0 +1 @@
+<%= render 'govuk_component/document_footer', @content_item.document_footer_for_new_navigation %>
```

Comment:
It looks like the only difference between templates are the parameters being passed to a component. Rather than switching templates, would it make more sense to use the presenter to alter the parameters passed to the component? This will prevent templates from getting out of sync.

Here is an example of a format overriding the default method:
https://github.com/alphagov/government-frontend/blob/f3700d94a92c318bae5032174d1e5ac2b2f4361a/app/presenters/speech_presenter.rb#L28-L41

Default defined here:
https://github.com/alphagov/government-frontend/blob/f3700d94a92c318bae5032174d1e5ac2b2f4361a/app/presenters/metadata.rb